src/assert_utils.js

```diff
@@ -101,6 +101,30 @@ function assertIncludes(haystack, needle, message = undefined) {
     );
 }
 
+/**
+* Assert that a string is <b>not</b> included in another, or object is <b>not</b> included in an array.
+*
+* @example
+* ```javascript
+* assertNotIncludes('foobar', 'xxx');
+* assertNotIncludes([9, 5, 3], 2);
+* ```
+* @param {string|array} haystack The thing to search in.
+* @param {string|array} needle The thing to search for.
```

Comment:
Is this signature correct? `array` for the needle sounds odd.

Reply:
Good point, wondering if the signature should use generics instead:

```js
/**
 * @template T
 * @param {T[]} haystack
 * @param {T} needle
 */
```

Reply:
Where T has an `includes` method 😅